<launch>
  <node pkg="rosbag" type="play" name="rosbag" required="true" args="/home/jaes/rosbag/2020-08-24-19-22-31.bag"/>
  <node name="extract" pkg="image_view" type="extract_images" respawn="false" required="true" output="screen" cwd="ROS_HOME">
    <remap from="image" to="/usb_cam/image_raw"/>
  </node>
</launch>
